content/articles/creation-of-calendar-heatmaps-to-display-time-series-quantitative-data-with-python/index.md

```diff
@@ -0,0 +1,115 @@
+﻿### Creation of Calendar Heatmaps to Display Time-Series Quantitative Data with Python
+In this  simple well-elaborated tutorial we have a lot of alternatives when it comes to choosing a visualization for our data.  
+For a visualization to be said to be  good it's not only easy to do but also has a cognitive and neural basis. A basic heatmap is a graphical representation of data in which individual values are represented by a color gradient.  
+Calendar heatmaps combine heatmaps and calendars into a single visualization. In a calendar view, a calendar heatmap employs colored cells to represent the relative quantity of events on each day.  
+Weeks divide the days into columns, which are then classified by months and years. This allows us to immediately spot daily and weekly patterns as well as to detect anomalies in the data.
+Time series is a design that collects data on the same variable at regular intervals(Days, Weeks, Months, Years).
+It allows the researcher to access the impact of treatment over time, making it easier to predict  future values based on previously observed values. Key areas that implement it are :
+- Business forecasting.
+- Understanding  previous behavior.
+- Planning the future.
+- Evaluate current accomplishments.
+### Table of contents
+-  [Prerequisites](#prerequisites)
+-  [Components of Time series](#components-of-time-series)
+-  [Installation of calmap Library](#installation-of-calmap-library)
+-  [Performing preprocessing from read data](#Performing-preprocessing-from-read-data)
+-  [Setting up the Calendar Plot with the Library Calplot Modules](#setting-up-the-calendar-plot-with-the-library-calplot-modules)
+- [Calendar Heatmap using the Python library ‘july’](#calendar-heatmap-using-the-python-library-‘july’)
+-  [Conclusion](#Conclusion)
+
+### Prerequisites
+1. Python Language.
+2. Be familiar working with CSV files, for reference click on  the link: [Working with CSV files in Python](/engineering-education/working-with-csv-files-in-python/)
+
+
+### Components of Time series
+Trends: it is a relative movement  of either higher or lower values for a very  long period of time, having an uptrend and a low trend.
+Seasonality: referred to as upward or downward swings always a repeating pattern for a fixed period of time
+Irregularity. known as 
+Cyclic: it has non-regular components having fluctuations within the trend.
+
+Without further ado lets jump into first things first. 
+The data we will be using in this tutorial is obtained from kaggle, to access it we can click on the link provided below.[Marketing Funnel by Olist | Kaggle](https://www.kaggle.com/olistbr/marketing-funnel-olist)
+Let's begin by first: 
+### Installation of calmap Library
+For us to be able to use calmap library, we first have to install it.  Below  is the command for its download and installation.
+```
+ pip install calplot
+```
```

Comment:
Because this is only one Python command, you can just use single backticks - ` `